spring-boot/src/main/java/org/springframework/boot/web/server/Ssl.java

```diff
@@ -101,6 +101,31 @@
 	 */
 	private String protocol = "TLS";
 
+	/**
+	 * Disable SSL compression.
+	 */
+	private Boolean disableCompression;
```

Comment:
Unfortunately I can't find some of these options in Jetty. I guess I'l just stick with a custom `EmbeddedServletContainerCustomizer` in my project.